index.js

```diff
@@ -61,18 +76,10 @@ function handleWin(regions) {
 function handleBoardClick(ev) {

```

Comment:
Uma estratégia legal que melhora muito a legibilidade/manutenção de códigos é o seguinte: você cria uma função de alto nível(com regras) que apenas vai chamar outras funções de mais baixo nível. Exemplo:
```js
function handleBoardClick(ev) {
  const row = getRow(ev)
  const column = getColumn(ev)

  fillElementWithPlayerSymbol(span, row, column)
  disableRegion(span)
  verifyIfThereIsAWinner()
  updateTitle()
}
```

Dessa maneira, você o core, as regras principais do seu négocio(um jogo da velha nesse caso). Se o desenvolvedor quiser saber como é feita um `disableRegion`, é só entrar no método. A implementação atual mistura detalhes de alto nível (como preencher uma célula apos o jogador a selecionar) com detalhes de baixo nível (como extrair a céula de um data set). Esse é um app relativamente simples mas em aplicações maiores esse tipo de estrutura dificulta muito o entendimento e manutenção.

Isso é o que o Uncle Bob chama no Clean code: "Don't mix different levels of abstraction".